src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
```

Comment:
do not apply any rules to the `*` selector except `box-sizing`

Reply:
done